src/main/scala/com/itechart/ny_accidents/constants/Configuration.scala

```diff
@@ -3,13 +3,32 @@ package com.itechart.ny_accidents.constants
 import com.typesafe.config.ConfigFactory
 
 object Configuration {
-  private val appConf = ConfigFactory.load("app.conf")
-  private val cacheConf = ConfigFactory.load("conf/cache.conf")
+  private lazy val appConf = ConfigFactory.load("conf/app.conf")
+  private lazy val cacheConf = ConfigFactory.load("conf/cache.conf")
+  private lazy val databaseConf = ConfigFactory.load("conf/ny_data_database.conf")
 
-  val CACHE_NAME: String = cacheConf.getString("cache.name")
-  val CACHE_PATH: String = "/home/" +
+  lazy val CACHE_NAME: String = cacheConf.getString("cache.name")
+  lazy val CACHE_PATH: String = "/home/" +
```

Comment:
To avoid string concatenation,  you could also use interpolation here

Reply:
Fixed